worldedit-core/src/main/java/com/sk89q/worldedit/EditSession.java

```diff
@@ -1732,6 +1733,87 @@ public int makeCylinder(BlockVector3 pos, Pattern block, double radiusX, double
         return affected;
     }
 
+    /**
+     * Makes a splatter.
+     *
+     * @param pos Center of the splatter
+     * @param block The block pattern to use
+     * @param radius The cylinder's radius
+     * @param decay The higher the number the more decay
+     * @return number of blocks changed
+     * @throws MaxChangedBlocksException thrown if too many blocks are changed
+     */
+    public int makeSplatter(BlockVector3 pos, Pattern block, double radius, int decay) throws MaxChangedBlocksException {
+        //  decay is 0 - 10, 0 should be a circle.
+        final Random random = new Random();
+
+        radius += 0.5;
+
+        int affected = 0;
+
+        if (decay <= 0) {
```

Comment:
could just as well be `< 0`.
these hard limits should probably also be noted in the param javadoc, and probably also `checkArgument`ed, at which point this clamping can be removed. (invisibly correcting mistakes leads to confusion, errors and documentation are explicit).